### ISSUE TITLE
Crosshair not centered on camera picture on Fairphone 2

### ISSUE THREAD

**Issue by ltog** [mapillary/mapillary_issues#2259]
### Basic information
- Platform: Android
- App Version: 2.39
- Device: Fairphone 2 (Fairphone Opensource OS version 16.07.1 (this is not the default OS with which Fairphone 2s are sold))
### Steps to reproduce behavior
1. Try to capture images using the Android app
### Expected behavior

The crosshair should be centered inside the camera image.
### Actual behavior

The crosshair is not centered inside the camera image, but instead in the middle of the app screen. (See screenshot)
### Additional information

Screenshot:

![mapillary-crosshair](https://cloud.githubusercontent.com/assets/4726277/18425613/45709052-78bb-11e6-97d2-d4db5bd0f0fd.png)

Sample image: http://mapillary.com/map/im/Zi-Omvys_y00LsWe7blB3g (I think this was captured with a slightly older version of the app, but definitely had the crosshair skewed.)

**Comment by HolgerJeromin** [comment]
Same Phone, same App Version, Fairphone stock rom (build date Aug 15; yes fairphone has monthly security updates :-)

![image](https://cloud.githubusercontent.com/assets/2410353/18426904/1eb1e600-78c5-11e6-888c-90c8c0907255.png)

**Comment by ltog** [comment]
Note that the crosshair ist NOT centered in the picture of @HolgerJeromin :

![crossed](https://cloud.githubusercontent.com/assets/4726277/18429911/5eb68718-78d6-11e6-9e1e-b7979a7c2401.png)

**Comment by ToeBee** [comment]
I've had this on my Galaxy S7 for a while now. I just never got around to reporting an issue. It is centered on the screen but not in the picture viewfinder.

![2016-09-14 04 38 20](https://cloud.githubusercontent.com/assets/348948/18501726/1e8cdd98-7a18-11e6-9736-7965303699c2.png)

**Comment by ltog** [comment]
FYI: This is still happening with version 321.

**Comment by HolgerJeromin** [comment]
What is the center of the images?
I found that the images are aspect ratio 16:9 -ish and the thumbnails (gwenview image viewer of KDE) was 4:3 with LARGER camera angle than in the image itself (more street and more sky visible). 
My Example was: Thumbnail showed traffic light above sky and the image only sky.

**Comment by jfburke3** [comment]
This appears to be directly related to the aspect ratio of the camera mode.  On my Galaxy S7 (4:3 is the largest resolution mode), I have the problems depicted above (i.e., crosshairs appear centered on the screen, but not the image).  On my Galaxy S5 (16:9 is the largest resolution mode), the image area takes up the whole screen, with the buttons overlaid on it, and the crosshairs are centered on the image (and also the screen).

In other words, the crosshairs as currently implemented center on the screen exclusively, regardless of the image area.

**Comment by ovivoda** [comment]
this issue should be fixed in the latest android app version
